bin/oneoff/data_fix/level_group_data

```diff
@@ -167,24 +167,47 @@ end
 # The main method for the script. Reads the CSV and persists UserLevel and
 # LevelSource objects as necessary.
 def main
+  user_level_ids = []
   CSV.foreach(DATA_FILE, headers: true, col_sep: "\t") do |row|
-    user_level_id = row['user_level_id'].to_i
-
-    user_level, level_source_data = get_user_level_and_level_source_data(
-      user_level_id
-    )
+    user_level_ids << row['user_level_id'].to_i
+  end
 
-    level_source_data = migrate_level_group_user_level user_level, level_source_data
+  puts "READ #{user_level_ids.count} IDs..."
+
+  e_user_level_id = nil
```

Comment:
A poorly named variable to hold the current `user_level_id` being processed, should an (e)xception occur.